druntime/mak/COPY

```diff
@@ -71,6 +71,7 @@ COPY=\
 	$(IMPDIR)\core\internal\elf\io.d \
 	\
 	$(IMPDIR)\core\internal\gc\bits.d \
+	$(IMPDIR)\core\internal\gc\blkcache.d \
```

Comment:
These should be part of the first commit

Reply:
These commits were split to aid in review. The first commit is a *straight* copy/paste of code. In essence, they are not separable -- you can't build with the first commit only, even if I were to make these changes go into the first commit.

I had a hard time doing the split, don't make me do it again...

Reply:
OK